### ISSUE TITLE
monitoring cd-rom images (iso9660) triggers space alerts

### ISSUE THREAD

**Issue by ssbarnea** [DataDog/dd-agent#1716]
I was surprised to see that the default settings of the datadog-agent do include monitoring of mounted cd-rom images... as these are read-only and already with 100% usage any general alerts would trigger false positive issues.

By default datadog should not monitor devices likes these (readonly or cdrom)
```
[root@zeus ~]# mount
/dev/sdb1 on / type ext3 (rw)
none on /proc type proc (rw)
none on /sys type sysfs (rw)
none on /dev/pts type devpts (rw)
none on /dev/shm type tmpfs (rw)
none on /sys/fs/cgroup type cgroup (rw,memory)
/opt/xxx/packages/iso/ccc.iso on /var/xen/xc-install type iso9660 (ro,loop=/dev/loop0)
sunrpc on /var/lib/nfs/rpc_pipefs type rpc_pipefs (rw)
/dev/mapper/XSLocalEXT--5ff69aec--4e28--6d7a--932f--4299ab4b7086-5ff69aec--4e28--6d7a--932f--4299ab4b7086 on /var/run/sr-mount/5ff69aec-4e28-6d7a-932f-4299ab4b7086 type ext3 (rw)
camos:/vol/tools on /var/run/sr-mount/6664706d-0afb-f77f-5570-74dd8040557b type nfs (rw,soft,timeo=600,retrans=2147483647,proto=tcp,acdirmin=0,acdirmax=0,addr=10.80.16.58)
filer02:/vol/groups/windev on /var/run/sr-mount/9088d2e9-29fc-5996-1bca-234cedd7228a type nfs (rw,soft,timeo=600,retrans=2147483647,proto=tcp,acdirmin=0,acdirmax=0,addr=10.80.16.175)
```

**Comment by remh** [comment]
Thanks @ssbarnea we'll take a pass at it for 5.5.0

**Comment by yannmh** [comment]
Should be fixed with https://github.com/DataDog/dd-agent/pull/1786.

**Comment by yannmh** [comment]
Closed with #1786.
